src/intervals/conversion.jl

```diff
@@ -31,16 +25,12 @@ function convert{T<:AbstractFloat, S<:Real}(::Type{Interval{T}}, x::S)
 end
 
 function convert{T<:AbstractFloat}(::Type{Interval{T}}, x::Float64)
-    II = convert(Interval{T}, rationalize(x))
-    # This prevents that rationalize(x) returns a zero when x is very small
-    if x != zero(x) && II == zero(Interval{T})
-        return Interval(parse(T, string(x), RoundDown),
-                        parse(T, string(x), RoundUp))
-    end
-    II
+    isinf(x) && return Interval{T}(x)
```

Comment:
Good point. Currently this was returning one of the edge intervals, e.g. `Interval(realmax(), Inf)`, which was handled at the constructor. The idea to include that here was to skip the rest of the method. (I will change the constructor as you suggest below, and the adapt this.)